src/api-reference/direct-connects/hotel-service-2/Update-History.markdown

```diff
@@ -0,0 +1,11 @@
+---
+title: Update History - Hotel Service 2
```

Comment:
For consistency with the other pages...
`title: Direct Connect - Hotel v2 - Update History`

Reply:
done